Sources/SwiftPolyglot/AsyncParsableCommand.swift

```diff
@@ -0,0 +1,5 @@
+import ArgumentParser
+
+protocol AsyncParsableCommand: ParsableCommand {
```

Comment:
Doesn't this already exist? https://apple.github.io/swift-argument-parser/documentation/argumentparser/asyncparsablecommand/

Reply:
Nice!! I didn't see it!! This simplifies a lot

The custom code is now being removed